kinto/core/resource/__init__.py

```diff
@@ -1128,6 +1128,9 @@ def _extract_filters(self, queryparams=None):
         filters = super(ShareableResource, self)._extract_filters(queryparams)
 
         ids = self.context.shared_ids
+        if getattr(self.context, "forced_empty_list", False):
+            ids = ['UNKNOWN']
```

Comment:
Maybe there is a better way here so that we don't even bother triggering a storage query since we want an empty list here.